quickstart.eve.md

```diff
@@ -21,11 +27,11 @@ commit
 ~~~
 ``````
 
-This record committed by this block is tagged `#greeting`, and has an attribute "message" with the value "hello world". Although tags have a special syntax, they are attributes like any other. We encourage you to classify groups of related records with tags.
+The record committed by this block is tagged `#greeting`, and has an attribute "message" with the value "hello world". Tags are attributes like any other, their special syntax notwithstanding. We encourage you to use tags to classify groups of related records.
```

Comment:
I think Rob was against this in an earlier discussion, since it muddied the waters.